README.md

```diff
@@ -103,6 +103,35 @@ assert('object' === type(new Object));
 assert('object' === type(new String('hello')));
 ```
 
+#### ECMA6 Types
+Supports all ECMA6 Types:
+```js
+asert('map' === type(new Map());
+assert('weakmap' === type(new WeakMap()));
+assert('set' === type(new Set()));
+assert('weakset' === type(new WeakSet()));
+assert('symbol' === type(Symbol()));
+assert('promise' === type(new Promise(callback));
+assert('int8array' === type(new Int8Array()));
+assert('uint8array' === type(new Uint8Array()));
+assert('uint8clampedarray' === type(new UInt8ClampedArray()));
+assert('int16array' === type(new Int16Array()));
+assert('uint16array' === type(new Uint16Array()));
+assert('int32array' === type(new Int32Array()));
+assert('uint32array' === type(new UInt32Array()));
+assert('float32array' === type(new Float32Array()));
+assert('float64array' === type(new Float64Array()));
+assert('arraybuffer' === type(new ArrayBuffer()));
+assert('dataview' === type(new DataView(arrayBuffer)));
+```
```

Comment:
Yoda-style is actually my personal preference (which is why it was in the docs like that), but it might read better here if it was back to normal. 

:+1:  for the change for all examples in the docs.